app/presenters/address/select.presenter.js

```diff
@@ -8,10 +8,33 @@
 /**
  * Formats data for the `address/{sessionId}/select` page
  *
+ * @param {object[]} addresses - An array of address objects returned from the address lookup
+ *
  * @returns {object} - The data formatted for the view template
  */
-function go() {
-  return {}
+function go(addresses) {
+  return {
+    addresses: _addresses(addresses),
+    pageTitle: 'Select the address',
+    postcode: addresses[0].postcode
+  }
+}
+
+function _addresses(addresses) {
+  const displayAddresses = addresses.map((address) => {
+    return {
+      value: address.uprn,
+      text: address.address
+    }
+  })
+
+  displayAddresses.push({
```

Comment:
Have this at the start of the options in the drop-down. I believe `unshift()` will at it to the start.

```suggestion
  displayAddresses.unshift({
```